source.js

```diff
@@ -76,3 +78,7 @@ function compileNamespaceStep(name) {
   name = camelCase(name);
   return 'g=(g.' + name + '||(g.' + name + ' = {}))';
 }
+
+function getDependencies(dependencies) {
+  return dependencies ? JSON.stringify(dependencies).replace(/^\[/g, "").replace(/]$/g, "") : "";
```

Comment:
@sprilukin You could just ensure that `dependencies` is an array, keep the brackets, and replace `[deps()]` with `deps()` in template.js.